boss-ws/src/main/java/org/genomebridge/boss/http/service/DatabaseBossAPI.java

```
@@ -251,6 +253,27 @@ else if ( req.httpMethod.equals(HttpMethod.GET) ||
         return null;
     }
 
+
+    @Override
+    public ErrorDesc getResumableUploadId(String objectId, String userName, MultiResponse resp) {
+        ErrorDesc error = null;
+        if ( userName == null  ){
+            error = badReqErr(getMessage("remoteUser"));
+        }
+        BossDAO dao = getDao();
+        if ( !dao.canWrite(objectId,userName) ) {
+            error = writePermsErr(objectId,userName);
+        }
+
+        ObjectRow rec = dao.findObjectById(objectId);
+        if ( rec == null ) {
+            error = notFoundErr(objectId);
```

Same here, should return the error.